### ISSUE TITLE
Alt text covers image

### ISSUE THREAD

**Issue by teroyks** [DagAgren/toot#554]
When viewing an image, the alt text is always shown. If it is long it obscures the image it describes.

My preferred solution would be to at least have an option to not show alt texts for images (unless maybe I tap on the displayed image) – I can see the image, I don't need the description by default.

**Comment by kayssun** [comment]
Yes. I like that it’s visible by default (great discoverability) but it should not cover most of the screen and maybe one should be about to swipe it away (down).
A setting to make it disappear completely might be encouraging people to not care about alt text. A setting should not be the only option to make an image visible under a long alt text.

**Comment by teroyks** [comment]
For me it’s the opposite – if the alt text is there always, even for those who don’t need or want it, it’s an incentive to leave it out when posting.

**Comment by teroyks** [comment]
> You can tap the alt text to hide it.

D’oh! Why didn’t I think of trying that! 🤦 (Just tried tapping the image to bring it to front, which closes it instead.)
Thanks!

(I’d still prefer to have the option for showing/hiding the text by default, but this is an acceptable workaround.)

**Comment by alexbayleaf** [comment]
Perhaps truncate it to a couple of hundred characters at most, with one tap to expand and a swipe to remove?  But honestly I don’t mind the current behaviour. I agree the discoverability is a great thing, and a single tap to get rid of it is no big deal.
